dnsmasq_interface.c

```diff
@@ -87,7 +87,7 @@ void FTL_new_query(unsigned int flags, char *name, struct all_addr *addr, char *
 	if(config.privacylevel >= PRIVACY_HIDE_DOMAINS_CLIENTS)
 	{
 		free(client);
-		client = strdup("0.0.0.0");
+		client = strdup("::");
```

Comment:
Is this intended?

Reply:
Yes, `0.0.0.0` is not really well-defined whereas `::` is officially the "unspecified address".